From e83834bb04912988938273a044cbed5ef3c322ee Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 22 Mar 2021 10:35:48 +0000
Subject: GCC: fix template specialization in ToV8Traits<IDLArray<T>>::ToV8

GCC complains that explicit specialization in non-namespace scope
is happening for ToV8Traits<IDLArray<T>>::ToV8. However, the
specialization isn't needed here so remove it.

Bug: 819294
Change-Id: I75082262853c5e395cdd81615152e176f7400176
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2777553
Reviewed-by: Yuki Shiino <yukishiino@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/master@{#865095}
---
 third_party/blink/renderer/bindings/core/v8/to_v8_traits.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h b/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h
index 6c79433ce946..0e1f469b2861 100644
--- a/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h
+++ b/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h
@@ -683,7 +683,6 @@ struct ToV8Traits<IDLArray<T>> {
   }
 
   // TODO(crbug.com/1185046): Remove this overload.
-  template <>
   static v8::MaybeLocal<v8::Value> ToV8(
       ScriptState* script_state,
       const Vector<v8::Local<v8::Value>>& value) WARN_UNUSED_RESULT {
-- 
2.25.1

